src/tactic/interactive_expr.lean

```diff
@@ -149,6 +149,24 @@ meta def goto_def_button {γ} : expr → tactic (list (html (action γ)))
       attr.val "title" "go to definition"] ["↪"]]
   ) <|> pure []
 
+/-- Due to a bug in the webview browser, we have to reduce the number of spans in the expression.
+Do do this, we collect the attributes from `sf.block` and `sf.highlight` after an expression boundary. -/
+meta def get_block_attrs {γ}: sf → tactic (sf × list (attr γ))
+| (sf.block i a) := do
+  let s : attr (γ) := style [
+    ("display", "inline-block"),
+    ("padding-left", "1ch"),
+    ("text-indent", "-1ch"),
+    ("white-space", "pre-wrap"),
+    ("vertical-align", "top")
+  ],
+  (a,rest) ← get_block_attrs a,
```

Comment:
A related question: does
`[style [("x", "y")], className "a", style [("z", "w")], className "b"]`
do the right thing?

Reply:
yes it does

Reply:
it does a merge of the styles

Reply:
and classNames

Reply:
So it's equal to `[style [("x","y"), ("z","w")], className "a b"]` when it gets emitted.